--- pTk/mTk/generic/tkSelect.c~	2007-05-05 20:40:42.000000000 +0200
+++ pTk/mTk/generic/tkSelect.c	2009-05-06 22:20:11.955893997 +0200
@@ -1767,7 +1767,7 @@
 	}
 	*((long *) buffer) = (long) infoPtr->time;
 	*typePtr = XA_INTEGER;
-	*formatPtr = 8*sizeof(long);
+	*formatPtr = 32;
 	return 1;
     }
 
@@ -1800,7 +1800,7 @@
 	    }
 	}
 	*typePtr = XA_ATOM;
-	*formatPtr = 8*sizeof(Atom);
+	*formatPtr = 32;
 	return (ap - ((Atom *) buffer));
     }
 
