--- Makefile.PL.orig	2018-08-17 14:14:49.831057217 +0200
+++ Makefile.PL	2018-08-17 14:17:19.704992380 +0200
@@ -37,9 +37,7 @@
 use ExtUtils::MakeMaker;
 use File::Basename;
 
-my $versionfile = "lib/PDF/API2/Version.pm";
-
-require "$versionfile";
+my $version = "0.73";
 
 my %makehash=( 
     NAME            => "PDF::API2", 
@@ -53,8 +51,8 @@
     MAN1PODS        => {}, 
     MAN3PODS        => {}, 
     PREREQ_PM       => { 'Compress::Zlib' => 1.0, }, 
-    VERSION_FROM    => $versionfile,
-    BINARY_LOCATION => q|PDF-API2-|.$PDF::API2::Version::CVersion{vPerl}.q|.ppm.tar.gz|,
+    VERSION         => $version,
+    BINARY_LOCATION => q|PDF-API2-|.$version.q|.ppm.tar.gz|,
 );
 
 if(-f dirname($0).'/tarhelper.pm')
@@ -65,8 +63,6 @@
 
 WriteMakefile( %makehash );
 
-print STDERR $PDF::API2::Version::CVersion{vWeb}."\n";
-
 exit;
 
 
