Index: Makefile.PL
===================================================================
--- Makefile.PL	(revision 227)
+++ Makefile.PL	(working copy)
@@ -160,7 +160,7 @@
   $config{DEFINE} .= " -DHAVE_EXSLT"
 }
 
-$config{LIBS}.=' '.$Config{libs};
+# $config{LIBS}.=' '.$Config{libs}; ## this should not be needed, if yes, at least change it to $Config{perllibs}!
 
 if ($DEBUG) {
   print "LIBS: $config{LIBS}\n";
